pkg/sr/api.go

```diff
@@ -348,42 +348,59 @@ func (cl *Client) Schemas(ctx context.Context, subject string) ([]SubjectSchema,
 	return schemas, firstErr
 }
 
-// CreateSchema attempts to create a schema in the given subject.
+// CreateSchemaID attempts to create a schema with a fixed ID and version ID in the given subject,
```

Comment:
Yeah I'm totally unconvinced on the name. I like your proposal of RegisterSchema. I was thinking PostSchema schema but Register makes more sense for what the return value is.